packages/datetime2/test/components/dateRangePicker3Tests.tsx

```diff
@@ -1332,6 +1332,33 @@ describe("<DateRangePicker3>", () => {
         });
     });
 
+    describe.only("outside days", () => {
+        it("visually hides outside days when contiguous months are shown, even if showOutsideDays is true", () => {
+            const { left } = render({ initialMonth: new Date(2015, Months.DECEMBER, 2), dayPickerProps: { showOutsideDays: true } });
+            assert.equal(left.findOutsideDays().first().getDOMNode().computedStyleMap().get("visibility")?.toString(), "hidden");
+        });
+
+        it("allows outside days to be shown for single month date range", () => {
+            const { left } = render({ initialMonth: new Date(2015, Months.DECEMBER, 2), singleMonthOnly: true, dayPickerProps: { showOutsideDays: true } });
+            assert.equal(left.findOutsideDays().first().getDOMNode().computedStyleMap().get("visibility")?.toString(), "visible");
+        });
+
+        it("does not allow outside days to be selected even when visible", () => {
+            const { left } = render({ initialMonth: new Date(2015, Months.DECEMBER, 2), singleMonthOnly: true, dayPickerProps: { showOutsideDays: true } });
+
+            assert.equal(left.findOutsideDays().first().getDOMNode().computedStyleMap().get("pointer-events")?.toString(), "none");
+
+            // simulated clicks appear to not respect pointerEvents style or else would test with below
```

Comment:
no activity on it but I found https://github.com/enzymejs/enzyme/issues/2340

open to suggestions for better ways to disable the interaction or better ways to test this